bin/setup

```diff
@@ -13,10 +13,13 @@ FileUtils.chdir APP_ROOT do
   # This script is idempotent, so that you can run it at anytime and get an expectable outcome.
```

Comment:
I just went down a rabbit hole of "Wait, is `expectable` a word?" Results inconclusive because I won't pay for an online OED subscription and I don't trust dictionary.com.